bpf/windows/process_monitor.c

```diff
@@ -90,47 +106,52 @@ get_scratch_space()
 	return scratch;
 }
 
-char __ebpf_go_platform[] SEC(".ebpf_go_platform") = "windows";
-
 SEC("process")
-int process_monitor(process_event_t *ctx)
+int ProcessMonitor(process_md_t *ctx)
 {
-	process_event process_info;
-
-	memset(&process_info, 0, sizeof(process_info));
+	if (ctx->operation == PROCESS_OPERATION_CREATE) {
+		struct process_create_info_t process_create_info;
+
+		memset(&process_create_info, 0, sizeof(process_create_info));
+		process_create_info.common.op = 5;
```

Comment:
could we use MSG_OP_EXECVE constant

Reply:
I am currently not including any Linux-side headers. Today the build system is complicated. Once it is simplified in a way that we can build the programs in bpf/windows folder directly, I'll include the `msg_types.h`.